src/util/request-manager.js

```diff
@@ -118,6 +123,12 @@ export default class RequestManager {
     if (opts.httpsProxy != null) {
       this.httpsProxy = opts.httpsProxy;
     }
+
+    if (opts.cafile != null && opts.cafile != '') {
```

Comment:
This is awesome! Would you mind adding a short comment here explaining what this does and how it works?